style.css

```diff
@@ -0,0 +1,48 @@
+* {
+  box-sizing: border-box;
+}
+
+html { font-size: 10px; }
+
+body {
+  font-family: 'Courier New', Courier, monospace;
+  font-size: 2rem;
+  line-height: 1;
+  text-align: center;
+  color: black;
+  margin: 0;
+  background-color: chartreuse;
+}
+
+h1 {
+  font-size: 4rem;
+  margin-bottom: 0;
+}
+
+#main {
+  margin: 30px 8px;
+  padding: 15px;
+  border-radius: 5px;
+  background: #eee;
+}
```

Comment:
Excellent work! I appreciate your attention to utilizing classes and IDs. However, it is important to emphasize the significance of incorporating responsive values in our CSS files. Instead of relying solely on fixed pixel values, it is advisable to employ relative values such as percentages. Consider the following example:
`margin: 15% 2%;
`